### PR TITLE
{tools}[GCCcore/12.2.0] MUSCLE3 v0.7.0, QCG-PilotJob v0.13.1, Kaleido v0.2.1, msgpack-c v6.0.0, PyZMQ v25.1.0 w/ Python 3.10.8

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/q/QCG-PilotJob/QCG-PilotJob-0.13.1-gfbf-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    ('Python', '3.10.8'),
+    ('SciPy-bundle', '2023.02'),
+    ('PyZMQ', '25.1.0'),
+    ('Kaleido', '0.2.1'),
+    ('statsmodels', '0.14.0'),
+    ('plotly.py', '5.13.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
```
Comment (verdurin) [inline easybuild/easyconfigs/q/QCG-PilotJob/QCG-PilotJob-0.13.1-gfbf-2022b.eb:21]: @SimonPinches I think this is the default now and you can remove it, hence the warning from the bot.

**Review by verdurin — DISMISSED** [review]

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 60 out of 61 (6 easyconfigs in total)
centos7.lan - Linux CentOS Linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/8a7700ce8a117c62de8cbb75fa0dca8b for a full test report.

File: easybuild/easyconfigs/q/QCG-PilotJob/QCG-PilotJob-0.13.1-gfbf-2022b.eb (hunk 1 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/q/QCG-PilotJob/QCG-PilotJob-0.13.1-gfbf-2022b.eb]: Thanks @verdurin, fixed now.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node3100.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/b35a8f593172c032288462235ab8234d for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Going in, thanks @SimonPinches!

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 7 out of 7 (5 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/6f7e7b651fa319430c567493fc584848 for a full test report.

**Pull request merged**